### PR TITLE
feat: [M3-7683] -  Disable Create Image button with tooltip text on empty state Landing Page for restricted users

### PR DESCRIPTION
**Pull request by harsh-akamai**
## Description 📝
To prevent unauthorized access to specific flows and provide clearer guidance, we aim to restrict entry to users without the required permissions.

Here, we are restricting users from creating new Image from the Empty State Landing Page when they do not have access or have read only access.

## Changes  🔄
List any change relevant to the reviewer.
- For restricted users:
  - Disabled Create Image Button on the Empty Landing Page

## Preview 📷
| Before  | After   |
| ------- | ------- |
|||
| 📷 | 📷 |

### Prerequisites
(How to setup test environment)

- Log into two accounts side by side:
  - An unrestricted admin user account: full access
  - A restricted user account (use Incognito for this)
    - Start with Read Only for everything

### Reproduction steps
- Landing:
  - Observe as restricted user, notice shows and you cannot create Images

### Verification steps
- After changes, observe tooltips are tailored to the action.

## As an Author I have considered 🤔

*Check all that apply*

- [x] 👀 Doing a self review
- [x] ❔ Our [contribution guidelines](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md)
- [x] 🤏 Splitting feature into small PRs
- [x] ➕ Adding a changeset
- [ ] 🧪 Providing/Improving test coverage
- [ ] 🔐 Removing all sensitive information from the code and PR description
- [ ] 🚩 Using a feature flag to protect the release
- [x] 👣 Providing comprehensive reproduction steps
- [ ] 📑 Providing or updating our documentation
- [ ] 🕛 Scheduling a pair reviewing session
- [ ] 📱 Providing mobile support
- [ ] ♿  Providing accessibility support

### CODE REVIEW

File: packages/manager/src/features/Images/ImagesLanding/ImagesLandingEmptyState.tsx [hunk 1]
```diff
@@ -15,11 +17,16 @@ import {
 export const ImagesLandingEmptyState = () => {
   const { push } = useHistory();
 
+  const isRestricted = useRestrictedGlobalGrantCheck({
```
Comment (jaalah-akamai) [inline packages/manager/src/features/Images/ImagesLanding/ImagesLandingEmptyState.tsx:20]: nit: I would suggest just making this more descriptive `isImagesReadOnly`

**Review by jaalah-akamai — APPROVED** [review]

File: packages/manager/src/features/Images/ImagesLanding/ImagesLandingEmptyState.tsx (hunk 1 above)
Comment (harsh-akamai) [reply packages/manager/src/features/Images/ImagesLanding/ImagesLandingEmptyState.tsx]: Made the required changes.

File: packages/manager/.changeset/pr-10670-tech-stories-1721055859684.md
Comment (mjac0bs) [inline packages/manager/.changeset/pr-10670-tech-stories-1721055859684.md:1]: We only need one changeset per PR, which summarizes the overall changes. During our release process, a [script](https://github.com/linode/manager/blob/develop/scripts/changelog/generate-changelogs.mjs) will compile these into our changelog, which we publish here: https://github.com/linode/manager/releases.

We can delete this changeset.

File: packages/manager/src/features/Images/ImagesLanding/ImagesLandingEmptyState.test.tsx
Comment (mjac0bs) [inline packages/manager/src/features/Images/ImagesLanding/ImagesLandingEmptyState.test.tsx:1]: Thanks for including test coverage!

**Review by mjac0bs — APPROVED** [review]
Thanks @harsh-akamai - the disabled change for restricted users and the test coverage looks good.

We'll just want to delete the most recently added changeset.

**Review by hkhalil-akamai — APPROVED** [review]
Great work Harsh!

File: packages/manager/.changeset/pr-10670-tech-stories-1721055859684.md
Comment (harsh-akamai) [reply packages/manager/.changeset/pr-10670-tech-stories-1721055859684.md:1]: Deleted the second changeset

File: packages/manager/src/features/Images/ImagesLanding/ImagesLandingEmptyState.test.tsx
Comment (harsh-akamai) [reply packages/manager/src/features/Images/ImagesLanding/ImagesLandingEmptyState.test.tsx:1]: No problem!

**Pull request merged**